package.json

```diff
@@ -23,8 +23,10 @@
     "format": "prettier --write \"**/*.{ts,tsx,md}\""
   },
   "devDependencies": {
+    "@types/file-saver": "^2.0.5",
```

Comment:
this branch is 'rebased' onto #102 hence these 'appconfig' changes. same with the genetics pr